GeneralsMD/Code/Libraries/Source/WWVegas/WWMath/matrix3d.cpp

```diff
@@ -520,7 +522,9 @@ void Matrix3D::Get_Inverse(Matrix3D & inv) const
 	Matrix4x4	mat4Inv;
 
 	float det;
+#ifdef _WIN32
 	D3DXMatrixInverse((D3DXMATRIX *)&mat4Inv, &det, (D3DXMATRIX*)&mat4);
+#endif
```

Comment:
I think some alternative implementation should be provided or some kind runtime error should be produced...